packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -836,14 +840,24 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         private _storage?: IDocumentStorageService,
     ) {
         super();
-
         this.baseSummaryMessage = metadata?.message;
         /**
           * gcFeature in metadata is introduced with v1 in the metadata blob. Forced to 0/disallowed before that.
           * For existing documents, we get this value from the metadata blob.
           * For new documents, we get this value based on the gcAllowed flag in runtimeOptions.
           */
         const prevSummaryGCVersion = existing ? getGCVersion(metadata) : undefined;
+
+        /**
+         * If this is a new container, we initialize these values. If it's an existing container,
+         * these values would stay undefined
+         */
+        if (!existing) {
+            this.createContainerMetadata.createContainerRuntimeVersion = pkgVersion;
```

Comment:
I might be missing it, but I do not see this.createContainerMetadata being populated anywhere else.
So if it's not existing, it seems like we never read it from snapshot? And then any logging would be only correct for the session that created it?

Reply:
sorry i missed that